content/en/guides/hosting/monitoring-usage/org_dashboard.md

```diff
@@ -3,62 +3,95 @@ menu:
-- **Activity**: lists details about each user's activity.
-- **Teams**: lists details about each team, including the number of users and tracked hours, and allows an admin to join a team.
-- **Billing**: ssummarizes your organization's charges, allows you to run and export billing reports, and shows details about your license, such as when it expires.
-- **Settings**: allows you to configure custom roles and settings related to privacy and authentication.
+{{< tabpane text=true >}}
+{{% tab header="Dedicated / Self-managed" value="dedicated" %}}
+1. To access the **Organization Dashboard**, navigate to `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default. It lists all users, along with data about each user.
+1. To sort the list by user status, click the **Last Active** column label. Each user's status is one of the following:
 
-## View the status of a user
-The **Users** tab lists all users, along with data about each user. The **Last Active** column shows whether a user has accepted an invitation and the user's current status:
+    * **Invite pending**: Admin has sent invite but user has not accepted invitation. 
+    * **Active**: User has accepted the invite and created an account.
+    * **-**: The user was previously active but has not been active in the last 6 months.
+    * **Deactivated**: Admin has revoked access of the user.
+1. To see details about a user's last activity, hover your mouse over the **Last Active** field for the user.  A tooltip appears that shows when the user was added and how many total days the user has been active.
 
-* **Invite pending**: Admin has sent invite but user has not accepted invitation. 
-* **Active**: User has accepted the invite and created an account.
-* **-**: The user was previously active but has not been active in the last 6 months.
-* **Deactivated**: Admin has revoked access of the user.
+    A user is _active_ if they:
+    - log in to W&B.
+    - view any page in the W&B App.
+    - log runs.
+    - use the SDK to track an experiment.
+    - interact with the W&B Server in any way.
+{{% /tab %}}
 
-To sort the list of users by activity, click the **Last Active** column heading.
+{{% tab header="Multi-tenant SaaS Cloud" value="saas" %}}
+1. Navigate to the [**Members** page](https://wandb.ai/account-settings/wandb/members/). This page lists all users, along with data about each user.
+1. To sort the list by user status, click the **Last Active** column label. Each user's status is one of the following:
 
-## View and share how your organization uses W&B
-From the **Users** tab, view details about how your organization uses W&B in CSV format.
+    * **Invite pending**: Admin has sent invite but user has not accepted invitation. 
+    * **Active**: User has accepted the invite and created an account.
+    * **-**: The user was previously active but has not been active in the last 6 months.
```

Comment:
Maybe add a space before the colon? (Or describe that it's a dash?) 

It looks like a mistake in the docs.
<img width="204" height="41" alt="Screenshot 2025-07-22 at 4 06 42 PM" src="https://github.com/user-attachments/assets/2d63d7ef-ecbe-4b1f-9fa6-7aecf4c5fe4d" />